src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -1900,11 +1897,11 @@ From there, simply click on one of the supported files in the File Explorer and
     <value>Customize the shortcut to turn on or off this mode</value>
     <comment>"Mouse Highlighter" is the name of the utility. Mouse is the hardware mouse.</comment>
   </data>
-  <data name="MouseUtils_MouseHighlighter_LeftButtonClickColor.Header" xml:space="preserve">
-    <value>Left button highlight color</value>
+  <data name="MouseUtils_MouseHighlighter_PrimaryButtonClickColor.Header" xml:space="preserve">
```

Comment:
Just as a double check: MouseHighlighter works as expected when turning left/right mouse buttons around, right :)?

Reply:
Up to now, the words are simply wrong. I think this is something you can easily test/experience yourself. :)